packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -134,6 +134,7 @@ class WebView extends StatefulWidget {
     Key key,
     this.onWebViewCreated,
     this.initialUrl,
+    this.initialPostParameters,
```

Comment:
I think we better start with a `WebViewController.postUrl` which covers the more generic use case (posting at any point). The `initialUrl` is a convenience mechanism for simple cases, I'd be hesitant to add complexity to it.

Reply:
Moved method to controller as suggested :)